extractor/event_extractor.go

```diff
@@ -81,12 +81,8 @@ func extractAuthzMessages(msg json.RawMessage) ([]json.RawMessage, error) {
 }
 
 func EventContextFromAuthz(ctx db.EventContext, msgIndex int) (db.EventContext, error) {
-	authzCtx := db.EventContext{
-		Batch:       ctx.Batch,
-		Timestamp:   ctx.Timestamp,
-		TxHash:      ctx.TxHash,
-		AuthzParent: &ctx,
-	}
+	authzCtx := ctx
```

Comment:
does this do a deep copy or will below ater the original ctx?

Reply:
The context itself is passed by value in function param, so original context won't be modified.